CHANGELOG.md

```diff
@@ -2,10 +2,20 @@
 
 ## v0.10.0
 
+### Technical improvements
+
 - Added new `recorder` module to build `Session Recorder` app
 - Added new `common` module to share code between `app` and `recorder` modules
 - Improved `VisionView` lifecycle management
 
+### User experience
+
+- MapboxVision v0.10.0
+- Added detection of construction cones
+- Improved quality of detection/segmentation, especially at night
+- Improved segmentation, now it's more focused on road specific elements. New segmentation model recognizes the following classes: Crosswalk, Hood, MarkupDashed, MarkupDouble, MarkupOther, MarkupSolid, Other, Road, RoadEdge, Sidewalk
+- General bug fixes
```

Comment:
I'd not add `general` point since there are real changes

Reply:
Will leave just `Bug fixes` then